include/zephyr/drivers/fuel_gauge.h

```diff
@@ -198,16 +171,8 @@ struct fuel_gauge_set_property {
 		/* type property_field; */
 
 		/* Writable Dynamic Battery Info */
-		/** FUEL_GAUGE_SBS_MFR_ACCESS */
-		uint16_t sbs_mfr_access_word;
-		/** FUEL_GAUGE_SBS_REMAINING_CAPACITY_ALARM */
-		uint16_t sbs_remaining_capacity_alarm;
-		/** FUEL_GAUGE_SBS_REMAINING_TIME_ALARM */
-		uint16_t sbs_remaining_time_alarm;
-		/** FUEL_GAUGE_SBS_MODE */
-		uint16_t sbs_mode;
-		/** FUEL_GAUGE_SBS_ATRATE */
-		int16_t sbs_at_rate;
+		/** See SBS spec for value interpretation */
+		uint16_t sbs_word;
```

Comment:
Adding here. But in a follow-up PR I plan on completely removing the set struct and just keeping a single struct.